.travis.yml

```diff
@@ -7,4 +7,4 @@ before_install:
 script:
 - docker run  todo_test_image tests/test_view_model.py
 - docker run --env SECRET_KEY=test_secret_key --env TRELLO_API_KEY=this_is_trello_api_key_123 --env TRELLO_TOKEN=and_this_is_the_trello_token_56789 --env BOARD_ID=5fb51906cc7d395b1b145b0e todo_test_image tests/test_integration.py
```

Comment:
You should be able to just use the .env.test file. instead of having this really long run command.

Reply:
Thanks.  Have modified `.travis.yml` file to use `.env.test` file instead of the long command

`WORKDIR /devops_mod5/todo_app/` has been deleted from `Dockerfile` as well